### PR TITLE
docs : Adding warning message for migration of ApplicationSet into Argo CD

### PR DESCRIPTION
**Pull request by rishabh625**
Edited Readme to display sign of work in progress

### CODE REVIEW

**Comment by rishabh625** [comment]
@wtam2018 please review and merge this

File: README.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Argo CD ApplicationSet Controller 
 
+### :warning: This project code has been moved to the main Argo CD repository
+
+This repository will no longer be active.It is now merged with ArgoCD and will be released along with it. Further development will happen in [Argo CD](https://github.com/argoproj/argo-cd).
```
Comment (wtam2018) [inline README.md:5]: ``
```suggestion
This repository is no longer active. ApplicationSet has been merged with Argo CD and will be released along with it. Further development will happen in [Argo CD](https://github.com/argoproj/argo-cd).
```

**Comment by wtam2018** [comment]
@jgwest @crenshaw-dev PTAL. I think we are ready to add a warning banner in this repo about moving the development to Argo CD.

**Review by crenshaw-dev — APPROVED** [review]
LGTM!

File: README.md [hunk 2]
```diff
@@ -1,5 +1,9 @@
 # Argo CD ApplicationSet Controller 
 
+### :warning: This project code has been moved to the main Argo CD repository
+
```
Comment (wtam2018) [inline README.md:4]:
```suggestion
## :warning: This project code has been moved to the main Argo CD repository

```

**Pull request merged**